Model/PaymentMethod.php

```diff
@@ -218,4 +243,41 @@ public function getSupportConfigFields()
 
         return $aFieldsPublic;
     }
+
+    /**
+     * Adds all needed data to the post-processing transaction
+     *
+     * @param Transaction $oTransaction
+     * @param Transaction $oParentTransaction
+     *
+     * @since 1.0.1
+     */
+    public function addNeededDataToTransaction(&$oTransaction, $oParentTransaction)
```

Comment:
Please rename to addPostProcessingTransactionData(), as in the comment you say its only for post processing.